tests/sentry/grouping/test_uniq_id_parameterization.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+
+from sentry.grouping.strategies.message import replace_uniq_ids_in_str
+
+
+@pytest.mark.parametrize(
+    ("name", "input", "expected"),
+    [
+        (
+            "Uniq ID - sql savepoint",
+            '''SQL: RELEASE SAVEPOINT "s140177518376768_x2"''',
+            """SQL: RELEASE SAVEPOINT <uniq_id>""",
```

Comment:
A lot of the test cases here are also in `tests/sentry/grouping/test_normalize_message.py`.

Why are we adding it again?